app/views/guidance/claim-guidance-for-farmers.njk

```diff
@@ -0,0 +1,104 @@
+      <button onclick="window.print()" class="govuk-link govuk-body-s gem-c-print-link__button">Print this page</button>
+    </div>
+
+    <div class="govspeak govuk-!-margin-top-6">
+      <h2 id="when-to-claim" class="govuk-heading-m" >When to claim</h2>
+  
+      <p class="govuk-body">You can claim funding after the vet has visited your farm to carry out the review and given you a written report and a summary of the review.</p>
+  
+      <p class="govuk-body">You must claim within 6 months from the date you accept your agreement offer.</p>
+  
+      <p class="govuk-body">If the review has not taken place and you need more time, contact the Rural Payments Agency (<abbr title="Rural Payments Agency">RPA</abbr>).</p>
+  
+      <p class="govuk-body">If the review has taken place and you have a good reason why you cannot make your claim within 6 months such as illness, contact the <abbr title="Rural Payments Agency">RPA</abbr> as soon as you can.</p>
+  
+      <h2 id="what-information-youll-need-to-claim" class="govuk-heading-m">What information you’ll need to claim</h2>
+      
+      <p class="govuk-body" >When you claim you’ll need to share the:</p>
+      
+      <ul class="govuk-list govuk-list--bullet">
+        <li>date of the vet’s last visit to the farm for the review</li>
+        <li>vet’s name</li>
+        <li>vet’s <abbr title="Royal College of Veterinary Surgeons">RCVS</abbr> number</li>
+        <li>unique reference number for laboratory test results</li>
+      </ul>
+  
+      <p class="govuk-body">The summary the vet will give you will include this information.</p>
+      
+      <h2 id="claim-funding" class="govuk-heading-m">Claim funding</h2>
+
+      <section class="gem-c-devolved-nations">
+        <h2 class="govuk-heading-s govuk-!-margin-bottom-0">
+          <p class="govuk-body">Find out <a rel="external" href={{ claimServiceUri }} class="govuk-link">how to claim funding</a>.</p>
+        </h2>
+     </section>
+      
+      <h2 id="get-help-with-your-claim" class="govuk-heading-m">Get help with your claim</h2>
+  
+      <p class="govuk-body">Contact the <abbr title="Rural Payments Agency">RPA</abbr> if you:</p>
+  
+      <ul class="govuk-list govuk-list--bullet">
+        <li>decide you do not want a review after you have accepted the agreement offer</li>
+        <li>cannot apply or claim online or have a question about your claim</li>
+      </ul>
+  
+      <p class="govuk-body">Have your agreement number with you when you call or include it in your letter or email.</p>
+
+      <div class="govuk-inset-text">
+        <p class="govuk-body">Rural Payments Agency<br>
+        PO Box 352<br>
+        Worksop<br>
+        S80 9FG<br>
+        </p>
+
+        {{ rpaContactDetails({email: ruralPaymentsAgency.email, callChargesUri: ruralPaymentsAgency.callChargesUri, telephone: ruralPaymentsAgency.telephone}) }}
```

Comment:
There is an additional blank line prior to the "Find out about call charges" link within the build that isn't in the prototype